### DISCUSSION TITLE
I need render the route content in a modal

### DISCUSSION THREAD

**Discussion by qq958691165** [inertiajs/inertia#1747] [Help]
As title, I use likely package [momentum-modal](https://github.com/lepikhinb/momentum-modal), but it seems not work very well.

**Comment by vitalijalbu** [comment]
I also want it, for example route login and registration I want a modal.
I did it by creating a custom layout with a modal, but it must be over the app layout 🧘😂

**Comment by RobertBoes** [comment]
Not sure what this has to do with Inertia, it's a third party package, so if that doesn't work you should create an issue on their repo

**Reply by vitalijalbu** [reply]
Right, but without using any packages, is that possible with inertia??

**Reply by RobertBoes** [reply]
Not easily no, since Inertia doesn't really supports modals/dialogs.

**Reply by qq958691165** [reply]
I know there is a issue [#642 ](https://github.com/inertiajs/inertia/pull/642) for 0.\*.\* about it, but it is no resolved in 1.\*.\*.

**Reply by vitalijalbu** [reply]
I did it manually, just created a popup layout (fillwidth) and use a normal page, it's not the best option but it works :D
